src/julia_to_gap.jl

```diff
@@ -24,6 +24,24 @@ julia> GAP.julia_to_gap([ [1, 2], [3, 4]], recursive = true)
 GAP: [ [ 1, 2 ], [ 3, 4 ] ]
 
 ```
+
+The following `julia_to_gap` conversions are supported by GAP.jl.
```

Comment:
What about the other direction? Are we actually "symmetric" in what we support? Might be good to state, too?

Reply:
Yes, of course the other direction makes sense. If you think the current proposal is more or less o.k. for `julia_to_gap` then I will update it to include also the other direction.